lib/main_common.pm

```diff
@@ -2759,12 +2774,14 @@ sub load_hypervisor_tests {
         loadtest "virtualization/universal/stresstest";           # Perform stress tests on the guests
         loadtest "console/perf";
         loadtest "console/oprofile" unless (get_var("REGRESSION", '') =~ /xen/);
+        loadtest "virtualization/universal/finish";               # Collect logs
     }
 
     if (check_var('VIRT_PART', 'windows')) {
         loadtest "virt_autotest/login_console";
         loadtest "virtualization/universal/download_image";       # Download Windows disk image(s)
         loadtest "virtualization/universal/windows";              # Import and test Windows
+        loadtest "virtualization/universal/finish";               # Collect logs
```

Comment:
Do you want to loadtest finish in all that matches unempty VIRT_PART? If yes, suggest to save multiple lines of this loadtest by putting below block in the end:
```
if ((get_var('VIRT_PART', '')) {
    loadtest "virtualization/universal/finish";
}
```

Reply:
BTW, what's the purpose of uploading logs when jobs succeed? The uploading logs function has been called in nearly every maintenance test module, which will take quite some test time in jobs that have long chained test modules. Is this considered and what is needed?

Reply:
This test takes **15s** with the SSH serial terminal and it's for debugging purposes.
Can you please recheck this now? I tried to consolidate loading of those modules.